src/sutta_publisher/shared/github_handler.py

```diff
@@ -0,0 +1,48 @@
+import json
+import logging
+from base64 import b64encode
+from io import BytesIO
+
+import requests
+
+log = logging.getLogger(__name__)
+
+
+def upload_file_to_repo(file_path: str, file: BytesIO, repo_url: str, api_key: str) -> None:
```

Comment:
Shouldn't we use `StringIO.StringIO` library instead of `io.BytesIO`? I'm not particularly familiar with this, so it's a question not a suggestion. I read in the article: [https://webkul.com/blog/using-io-for-creating-file-object/](https://webkul.com/blog/using-io-for-creating-file-object/):
> (...)
> * io.StringIO requires a Unicode string.
> * io.BytesIO requires a bytes string.
> * StringIO.StringIO allows either Unicode or Bytes string.
> * cStringIO.StringIO requires a string that is encoded as a bytes string.
> (...)

So it would seem that `StringIO.StringIO` is the more universal solution and doesn't require opening non-binary files as binaries.